code/testapp/src/main/java/com/adobe/marketing/optimizeapp/OffersScreen.kt

```diff
@@ -267,7 +266,7 @@ fun TextOffers(offers: List<Offer>? = null, placeholder: String = "Placeholder T
     listState?.also {
         LaunchedEffect(it) {
             snapshotFlow { listState.layoutInfo.visibleItemsInfo.map { lazyListItemInfo -> lazyListItemInfo.key } }
-                .map { visibleItemKeys -> visibleItemKeys.contains(offers?.get(0)?.proposition?.scope ?: "") }
+                .map { visibleItemKeys -> visibleItemKeys.contains(offers?.get(0)?.propositionReference?.get()?.scope ?: "") }
```

Comment:
Having to make changes in this class is an indication that you have changed the public APIs of Offer class. You should ensure this runs as-is without needing any changes